test/array.carp

```diff
@@ -191,6 +191,10 @@
                 "[(Pair 0 @\"a\") (Pair 1 @\"b\") (Pair 2 @\"c\")]"
                 &(str &(Array.enumerated &[@"a" @"b" @"c"]))
                 "enumerated works as expected")
+  (assert-equal test
+                "[]"
+                &(str &(Array.enumerated &(the (Array Int) [])))
```

Comment:
You should be able remove stringification here and compare the arrays directly by ref.

Reply:
Just copy pasted the test above but you're right.